app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009-2014 Martin Szulecki
+	2008-2010 Jonathan Beck
+	2009 Nikias Bassen
+	2008 Matt Colyer"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/libplist-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0effdedcb3de128c4930d8c03a3854c74c426c16728b8ab5f0a5b6bdc0b644be"
+
+ARCHITECTURES="x86 !x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+    commandBinDir=$binDir
```

Comment:
Oops, I did use tabs in my comments, but when you copy/paste them the tabs become spaces. It seems the only way preserve the tabs when you copy the blocs from my comments is to:
* click on the "edit" icon (i.e. the little pencil) to pretend you want to edit my comment;
* select and copy the block;
* click cancel;
* paste the clipboard to your editor.

Of course you can also replace each group of 4 spaces by a tab.

What I usually do before commiting is:
* look for 2 consecutive spaces anywhere;
* look for space+tab or tab+space anywhere;
* look spaces or tabs at end of lines.

Reminder: we try to always use tabs for the indentation. Of course, there may be exceptions from time to time, but here we prefer to have tabs. Sorry, BTW; I should have mentioned it earlier.